src/cli.coffee

```diff
@@ -35,6 +35,7 @@ watchCmd = program
   .option('-p, --port [port]', 'if a `server` option was specified, define on which port
  the server would run')
   .option('-d, --debug', 'print verbose debug output to stdout')
+  .option('-l, --stdin', 'listen to stdin and shut down when stdin closes')
```

Comment:
@es128 - It seems like the `-` option works fine, but then I saw this elsewhere:

```
10:51 AM shreeve@ice ~/Data/Work/Code/xo-next> bws -
Error: ENOENT, no such file or directory
  at Error (native)
  at process.chdir (/Users/shreeve/Data/Work/Code/brunch/node_modules/chokidar/node_modules/readdirp/node_modules/graceful-fs/polyfills.js:18:9)
  at module.exports.watch (/Users/shreeve/Data/Work/Code/brunch/src/watch.coffee:587:15)
  at start (/Users/shreeve/Data/Work/Code/brunch/src/index.coffee:9:3)
  at Command.listener (/Users/shreeve/Data/Work/Code/brunch/node_modules/commander/index.js:289:8)
  at Command.emit (events.js:110:17)
  ...
```

Is this easy to cleanup? If so, then the `-` option seems preferable to `-l`.